crates/plugin_css/src/dep_analyzer.rs

```diff
@@ -70,7 +70,6 @@ impl Visit for DepAnalyzer {
 pub fn is_source_ignored(source: &str) -> bool {
   source.starts_with("http://")
     || source.starts_with("https://")
-    || source.starts_with('/')
```

Comment:
we cannot just remove this condition. for url(/logo.png), the resolver may throw 'can not resolve' error, but the image may exist under public dir, in this situation /logo.png should not be handled.

if think we should add more judgement in  resolve hook of plugin css, try alias resolve first but external unresolved /logo.png

Reply:
Yes, I am trying to use the configuration in alias to determine whether further processing is needed.